share/picongpu/examples/atomicPhysics/include/picongpu/param/ionizer.param

```diff
@@ -0,0 +1,569 @@
+ * of the periodic table. The elements here should have a matching list of
+ * ionization energies in @see ionizationEnergies.param. Moreover this file
+ * contains a description of how to configure an ionization model for a species.
+ *
+ * Furthermore there are parameters for specific ionization models to be found
+ * here. That includes lists of screened nuclear charges as seen by bound
+ * electrons for the aforementioned elements as well as fitting parameters of
+ * the Thomas-Fermi ionization model.
+ */
+
+#pragma once
+
+#include "picongpu/particles/ionization/byCollision/collisionalIonizationCalc.def"
+#include "picongpu/particles/ionization/byCollision/ionizers.def"
+#include "picongpu/particles/ionization/byField/fieldIonizationCalc.def"
+#include "picongpu/particles/ionization/byField/ionizers.def"
+
+namespace picongpu
+{
+    /** Ionization Model Configuration
+     *
+     * - None : no particle is ionized
+     * - BSI : simple barrier suppression ionization
+     * - BSIEffectiveZ : BSI taking electron shielding into account via an effective
+     *                   atomic number Z_eff
+     * - ADKLinPol : Ammosov-Delone-Krainov tunneling ionization (H-like)
+     *               -> linearly polarized lasers
+     * - ADKCircPol : Ammosov-Delone-Krainov tunneling ionization (H-like)
+     *                -> circularly polarized lasers
+     * - Keldysh : Keldysh ionization model
+     * - ThomasFermi : statistical impact ionization based on Thomas-Fermi
+     *                 atomic model
+     *                 Attention: requires 2 FieldTmp slots @see memory.param
+     *
+     * Research and development:
+     * - BSIStarkShifted : BSI for hydrogen-like atoms and ions considering the
+     *                     Stark upshift of ionization potentials
+     *
+     * Usage: Add flags to the list of particle flags that has the following structure
+     *
+     *        ionizers< MakeSeq_t< particles::ionization::IonizationModel< Species2BCreated > > >,
+     *        atomicNumbers< ionization::atomicNumbers::Element_t >,
+     *        effectiveNuclearCharge< ionization::effectiveNuclearCharge::Element_t >,
+     *        ionizationEnergies< ionization::energies::AU::Element_t >
+     */
+    namespace ionization
+    {
+        /*! Specify (chemical) element
+         *
+         * Proton and neutron numbers define the chemical element that the ion species
+         * is based on. This value can be non-integer for physical models taking
+         * charge shielding effects into account.
+         * @see http://en.wikipedia.org/wiki/Effective_nuclear_charge
+         *
+         * It is wrapped into a struct because of C++ restricting floats from being
+         * template arguments.
+         *
+         * Do not forget to set the correct mass and charge via
+         * `massRatio<>` and `chargeRatio<>`!
+         */
+        namespace atomicNumbers
+        {
+            /** H-1 99.98% NA */
+            struct Hydrogen_t
```

Comment:
same as my last comment, please add the new materials to the original file or remove all other energies you do  not need for this example.